pkg/minikube/constants/constants_darwin.go

```diff
@@ -1,22 +0,0 @@
-//go:build (darwin && ignore) || !gendocs
```

Comment:
Why are these files deleted ?

Reply:
The only constant in this file was the one I removed.